src/lib/broadcast.jl

```diff
@@ -128,7 +128,7 @@ _broadcast(f::F, x...) where F = materialize(broadcasted(f, x...))
 
 _get(x::Tuple, i) = x[i]
 _get(::Nothing, i) = nothing
-collapse_nothings(xs::Vector{Nothing}) = nothing
+collapse_nothings(xs::Union{Vector{Nothing},Array{Nothing}}) = nothing
```

Comment:
Does this need the union, or could it just have `Array{Nothing}`?

Reply:
Oh, sorry. I don't recall the case for which I thought `Vector` isn't a Subtype of `Array`. I probably was wrong.

I agree, `AbstractArray{Nothing}` sound better.